### ISSUE TITLE
"Remember playback position" is gone???

### ISSUE THREAD

**Issue by RafaelLinux** [zaps166/QMPlay2#682]
Hi

In threads like https://github.com/zaps166/QMPlay2/issues/596 or https://github.com/zaps166/QMPlay2/issues/156 supposedly there were an option "Remember playback position". I never was able to use that option, cause QMPlay2 seems to ignore that if user doesn't open local file from "File, ....", that's, will not work if user click a video and QMPlay2 is opened by o.s. cause that media file is associated to QMPlay2.

However, I checked that option doesn't even exist yet in my version  

![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/817ab651-8e19-45ac-b32b-0a3d96be9757)

![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/cd2dff9e-9665-4a9a-9c73-b2890166edf0)

**Comment by zaps166** [comment]
Hey, this option is in "Playback settings" (3rd) tab :smile: 
Also there's new option at the end of the list on 3rd tab.
Sorry for mess in settings :sweat_smile:

**Comment by RafaelLinux** [comment]
That is indeed where the option is now, thank you.

![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/8fcaa60d-9f43-40cc-92be-09814072a644)

However, it still does not work as users expect. I don't usually work with video playlists. I simply - like most users I know - click on the video and it automatically opens it with QMPlayer2, because that's the video player I set up to play videos. And when I close QMPlayer while I'm playing a video that hasn't finished, I expect that when I click on the video again and QMPlayer2 starts playing it again, it will jump to the position where it was playing the last time. But it doesn't work like that for me. It just starts the video from the beginning every time.

**Comment by zaps166** [comment]
> But it doesn't work like that for me. It just starts the video from the beginning every time.

You have a button whether to continue playback or not (IIRC like in VLC). But if the video is too short (like 3 min), it'll not appear.

**Comment by zaps166** [comment]
![Screenshot_20240301_202244](https://github.com/zaps166/QMPlay2/assets/6239090/a0049e32-9fce-4dd4-afe5-8d2e2b473ef9)

**Comment by RafaelLinux** [comment]
Is it showed in compact mode??? As you can see, it's not showed despite I
enabled functionality and opened a closed video.

**Comment by zaps166** [comment]
> Is it showed in compact mode???

Yes.

**Comment by zaps166** [comment]
Idk why it doesn't work for you. Make sure this option is enabled.
Start long video, seek to the middle, close player, open video again. Button must be visible.

**Comment by RafaelLinux** [comment]
It's not a bad idea, but in my case, never worked!!!!!!!

**Comment by zaps166** [comment]
That's strange, sure you have this option enabled and your file has 8 min or more and you play within [1% - 99%]?

Example for 10 min file: it will ask to continue when you have slider between 0 min 6 sec and 9 min 54 sec.

![Zrzut ekranu_20240329_174224](https://github.com/zaps166/QMPlay2/assets/6239090/07329180-e868-41fb-a4f7-ae1d1129b05c)

**Comment by RafaelLinux** [comment]
As I explained in https://github.com/zaps166/QMPlay2/issues/682#issuecomment-1967829248 , is not usual to use playlist for videos. Is there a reason to implement it that way?

**Comment by zaps166** [comment]
do you have the button to continue playback as explained in https://github.com/zaps166/QMPlay2/issues/682#issuecomment-1973784543? Currently just a button without icon and logical placement in UI, but I see it works same as in VLC.

**Comment by RafaelLinux** [comment]
I never saw that button in QMplayer.
![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/c6607f4b-c8d8-42a7-a5ac-50a0619c92f1)

**Comment by zaps166** [comment]
Weird. Are you able to compile from sources and add one line of code?

Btw. I've changed button to icon and changed position.

**Comment by zaps166** [comment]
Anyway, I fixed few things, are you able to compile from master branch and test?

**Comment by RafaelLinux** [comment]
I could try if there are easy steps to do it.

**Comment by zaps166** [comment]
On Arch Linux (and its derivatives) you can use AUR client to build, e.g. `yay -S qmplay2-git`, on others probably there's no easy way.

**Comment by zaps166** [comment]
Released!

**Comment by RafaelLinux** [comment]
Yes!!! it's available yet in openSUSE Tumbleweed.

![image](https://github.com/zaps166/QMPlay2/assets/13048959/5ff51b16-389f-40ad-a69f-726160a71dc9)

However, nothing changed. That option you mentioned is nowhere in QMPlay2 window.

![image](https://github.com/zaps166/QMPlay2/assets/13048959/d0af381f-6222-4321-8124-7e853b00f93a)

**Comment by zaps166** [comment]
This button is moved and changed to icon, it has `C` hotkey. It's on the left to the slider, next to fullscreen button.
Also available in menu: `Player->Continue playback`.

**Comment by zaps166** [comment]
If it still not working - try [this](https://github.com/zaps166/QMPlay2/releases/download/24.04.02/QMPlay2-24.04.02-1-x86_64.AppImage).

**Comment by RafaelLinux** [comment]
I trying that version. Same result and interface  :(

**Comment by zaps166** [comment]
I don't know, I checked it on Windows, Linux, macOS - everywhere it's working also with YouTube links.
Check `$HOME/.config/QMPlay2` or `$HOME/.qmplay2` - you should have `UrlPos.ini` there with stored positions - do you have it?

**Comment by RafaelLinux** [comment]
I checked it. There is only one file with that name. It's under /home/user/.config/QMPlay2/Profiles/Nuevo/UrlPos.ini
I have another  two profiles in QMPlay2, however they have not any UrlPos.ini inside.
I changed now from one profile to another (from "Nuevo" to "Predefinido"  - is like "Default") but no changes about interface.

**Comment by zaps166** [comment]
Is `UrlPos.ini` empty? 

Do this test:
- play long video
- seek to the middle of the video
- press stop
- press play
- press "C" button (aka "Continue playback"), also watch the button next to the fullscreen button on the bottom toolbar.

**Comment by RafaelLinux** [comment]
Pressing  "C" jumped to the latest playback position, even simply "closing" QMPlay2 window while playing!!!! So it works fine, but unfortunately **still no a single button referencing that option**. I even don't see necessary a button to enable/disable it, but a icon showing that option is enabled.

**Comment by zaps166** [comment]
If "C" key is working, you should be able to click the button next to fullscreen button.

**Comment by RafaelLinux** [comment]
Yes, I see a new icon now.  But I would like to suggest two things:
- Change the icon to a less confusing one, like the one attached.
- Allow the user to enable automatic skip when replaying the video.

![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/f9dbf0ff-4324-4d4d-97c3-8480c7edd742)

Thanks in advance!!!

**Comment by zaps166** [comment]
What the automatic skip means?
Your icon is just refresh/reload icon. Also it depends on icon theme, every icon theme is different.

**Comment by RafaelLinux** [comment]
Sorry for the confusion. I explained myself badly. I mean that if I reopen a video (a movie) automatically, it jumps to the point where I left off to continue watching the movie, without having to press the "skip to last playback point" button. That's how other players do it and it's very convenient.

Regarding the icon, right now I see the ">" icon, which is a bit confusing. That's why my proposal to change it. I tried to find the right I was looking for, but I made a similar version cause I didn't find it.

![imagen](https://github.com/zaps166/QMPlay2/assets/13048959/36bb1711-4713-4d46-a1b6-dc180f8a6deb)
